examples/api/src/index.ts

```diff
@@ -52,9 +59,29 @@ async function asyncMain() {
     locale: process.env.MONGO_LOCALE ?? 'en'
   })
 
+  const oauth2Providers: Oauth2Provider[] = [
+    {
+      name: 'google',
+      discoverUrl: 'https://accounts.google.com',
+      clientId: '617896178757-i6ldn0nni9qtle8o6eu76lv93d78nvfi.apps.googleusercontent.com',
+      clientKey: 't267ZLqkV9dacrkPQp_pF-G2',
+      redirectUri: ['http://localhost:4000/auth/google'],
+      scopes: ['openid profile email']
+    },
```

Comment:
Get `clientId` and `clientKey` from enviroment and put defaults into `nodemon.json`.